File: python/1010_calculo_simples.py
_, N1, V1 = input().split()
N1 = int(N1)
V1 = float(V1)

_, N2, V2 = input().split()
N2 = int(N2)
V2 = float(V2)

VT = (N1 * V1) + (N2 * V2)

print(f'VALOR A PAGAR: R$ {VT:.2f}')

File: java/javascript/1 - Iniciante/1010_calculo_simples.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let [codigo_peca_1, num_pecas_1, valor_pecas_1] = lines[0].split(" ");
codigo_peca_1 = parseInt(codigo_peca_1);
num_pecas_1 = parseInt(num_pecas_1);
valor_pecas_1 = parseFloat(valor_pecas_1);

let [codigo_peca_2, num_pecas_2, valor_pecas_2] = lines[0].split(" ");
codigo_peca_2 = parseInt(codigo_peca_2);
num_pecas_2 = parseInt(num_pecas_2);
valor_pecas_2 = parseFloat(valor_pecas_2)

const total_peca_1 = num_pecas_1 * valor_pecas_1;
const total_peca_2 = num_pecas_2 * valor_pecas_2;
const total = total_peca_1 + total_peca_2;

console.log(`VALOR A PAGAR: R$ ${total.toFixed(2)}`);